src/services/swap_service.ts

```diff
@@ -82,6 +82,8 @@ import { serviceUtils } from '../utils/service_utils';
 import { SlippageModelManager } from '../utils/slippage_model_manager';
 import { utils } from '../utils/utils';
 
+const DEFAULT_SLIPPAGE_PERCENTAGE = 0.0005; // Default slippage rate if no `slippagePercentage` is specified by users
```

Comment:
If no API param is passed in the default is [here](https://github.com/0xProject/0x-api/blob/master/src/constants.ts#L29). Where 1 represents 100% slippage.

It may already be defaulted to this value in the `handler`, can't recall precisely.